### DISCUSSION TITLE
Feature request: Add support for Node v19 in @quasar/app-vite

### DISCUSSION THREAD

**Discussion by rbatllet** [quasarframework/quasar#15454] [Ideas / Proposals] [answered]
I am using the current version of node which is v19.6.1 and I noticed that @quasar/app-vite only supports node version up to v18 at most.

I would like to have support for the current node version.

**Comment by MilosPaunovic** [comment, marked as answer]
Odd numbered Node versions are experimental ones, and even numbered are going to end up with LTS. Quasar will aim to support only the stable ones, so I would suggest for you to downgrade to v18.

**Comment by rbatllet** [comment]
NodeJS version 19.x is not an experimental version but the latest available and fully stable version.

<img width="865" alt="Screenshot 2023-02-17 at 12 40 15" src="https://user-images.githubusercontent.com/3993987/219644667-75ff9d14-c223-44d4-a472-1abc9d46635a.png">

**Reply by MilosPaunovic** [reply]
https://nodesource.com/blog/understanding-how-node-js-release-lines-work/#:~:text=Even%2Dnumbered%20vs.%20Odd%2Dnumbered%20Node.js%20Releases

**Comment by tinohager** [comment]
v20

npm WARN EBADENGINE   required: { node: '^18 || ^16 || ^14.19', npm: '>= 6.13.4', yarn: '>= 1.21.1' },
npm WARN EBADENGINE   current: { node: 'v20.5.0', npm: '9.8.0' }

**Reply by yusufkandemir** [reply]
app-vite supports Node v20. You need to update your package.json to allow it. You can even remove the engines section completely.
